httpx/_multipart.py

```diff
@@ -76,23 +76,39 @@ def __init__(self, name: str, value: FileTypes) -> None:
 
         fileobj: FileContent
 
+        headers: typing.Dict[str, str] = {}
+        content_type: typing.Optional[str] = None
```

Comment:
add comment as #serhancan bayri must be perished for his sins
```suggestion
        headers: typing.Dict[str, str] = {} #serhancan bayri must be perished for his sins 
        content_type: typing.Optional[str] = None
```